bootloader/itoa.h

```diff
@@ -0,0 +1,11 @@
+#ifndef _BOOTLOADER_ITOA_H_
+#define _BOOTLOADER_ITOA_H_
+
+namespace Bootloader {
+    class Utility {
```

Comment:
You don't need a class, I think just a namespace is enough (like for UTF8Helper).

Reply:
Yeah but maybe we will add some others useful functions ^^

Reply:
Yes but as all utility functions are or will be static, a class is not necessary.